### DISCUSSION TITLE
Can't get stashing to work

### DISCUSSION THREAD

**Discussion by Styrke** [l10nelw/winger#33] [General]
I just discovered Winger and I am in love. ❤️ However, I have some challenges with the stashing feature.

When I try to stash a window by typing /stash in the omnibox, the window is closed, but the tabs don't appear to be saved as bookmarks.

I have enabled the stash feature in Winger's settings and granted Winger permission to manage bookmarks.
Currently on Firefox 119 on MacOS 14.1 with Winger version 2.4.0.

Any solutions or suggestions of things to try would be greatly appreciated. 🙏

**Comment by l10nelw** [comment]
Could you open `about:devtools-toolbox?id=winman%40lionelw&type=extension` and then do a stash? Please share what you see.

**Reply by Styrke** [reply]
Thank you for your reply.

Good news: I just went to try stashing with the developer console open, and this time it worked. Both stashing and unstashing. Unfortunately, I guess that means that it will be difficult to figure out why it didn't work for me originally.

However, another (minor) issue I see now is that the location where stashed windows get saved hasn't changed after picking a new location in the preferences. I did see an error in the console, though I don't know if it is related to any of this:
<img width="1624" alt="Screenshot 2023-11-14 at 10 24 23" src="https://github.com/l10nelw/winger/assets/1035167/c8e35ed8-b2d3-4c76-86c8-63d760b58a58">
